test/LRUCacheTest.cpp

```diff
@@ -28,6 +28,26 @@ TEST(LRUCacheTest, testTypicalUsage) {
   ASSERT_FALSE(cache["non-existant"]);
 }
 // _____________________________________________________________________________
+TEST(LRUCacheTest, testSimpleMapUsageWithDrop) {
+  LRUCache<string, string> cache(3);
+  cache.insert("1", "x");
+  cache.insert("2", "xx");
+  cache.insert("3", "xxx");
+  cache.insert("4", "xxxx");
+  cache.insert("5", "xxxxx");
+
+  ASSERT_FALSE(cache["2"]);         // second oldest dropped
+  ASSERT_FALSE(cache["1"]);         // oldest dropped
+  ASSERT_EQ(*cache["5"], "xxxxx");  // not dropped
+  ASSERT_EQ(*cache["4"], "xxxx");   // not dropped
+  ASSERT_EQ(*cache["3"], "xxx");    // not dropped
+  cache.insert("6", "xxxxxx");
+  ASSERT_FALSE(cache["5"]);          // oldest access driooed
```

Comment:
Typo in comment

Reply:
good find. This is in fact also in the branch I'd like to still finnish.